recipes/stringzilla/all/test_package/test_package.cpp

```diff
@@ -49,4 +49,27 @@ int main(void) {
   return EXIT_SUCCESS;
 }
 
+#else
+
+int main(void) {
```

Comment:
This test package is starting to get a little complicated but I don't know if there's anything to be done about that, as there's little duplication between the versions. Perhaps just have one function call per major version?

Reply:
fixed.